Common/File/FileUtil.cpp

```diff
@@ -708,7 +807,14 @@ bool DeleteDirRecursively(const Path &directory) {
 	return File::DeleteDir(directory);
 }
 
-void OpenFileInEditor(const Path &fileName) {
+bool OpenFileInEditor(const Path &fileName) {
+	if (Android_IsContentUri(fileName.ToString())) {
```

Comment:
Check type, REPORT?

-[Unknown]

Reply:
Not really worth a report. Adding proper type checking.